Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -133,12 +133,23 @@ YUI.add('ez-relation-editview', function (Y) {
          * @param {EventFacade} e
          */
         _runUniversalDiscovery: function (e) {
+            var that = this;
```

Comment:
niptick: please avoid using `that`, and bind the function given to `isSelectable`.